src/project/qml/MuseScore/Project/ProjectPropertiesDialog.qml

```diff
@@ -0,0 +1,317 @@
+                    }
+
+                    navigation.name: model.propertyName + "PropertyName"
+                    navigation.panel: propertiesListView.navigationPanel
+                    navigation.column: 3
+                    navigation.row: model.index
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    currentText: model.propertyValue
+                    hint: model.isStandardProperty ? "" : qsTrc("project", "Value")
+                    onCurrentTextEdited: function(newTextValue) {
+                        model.propertyValue = newTextValue
+                    }
+
+                    navigation.name: model.propertyName + "PropertyValue"
+                    navigation.panel: propertiesListView.navigationPanel
+                    navigation.column: 1
+                    navigation.row: model.index
+                    navigation.onActiveChanged: {
+                        if (navigation.active) {
+                            propertiesListView.positionViewAtIndex(index, ListView.Contain)
+                        }
+                    }
+                }
+
+                FlatButton {
+                    Layout.preferredWidth: 30
+                    Layout.preferredHeight: 30
+                    icon: IconCode.DELETE_TANK
+                    opacity: !model.isStandardProperty
+                    enabled: !model.isStandardProperty
+                    onClicked: projectPropertiesModel.deleteProperty(model.index)
+
+                    navigation.name: model.propertyName + "Delete"
+                    navigation.panel: propertiesListView.navigationPanel
+                    navigation.column: 2
+                    navigation.row: model.index
+                }
+            }
+        }
+
+        Connections {
+            target: projectPropertiesModel
+
+            function onPropertyAdded(index) {
+                propertiesListView.positionViewAtIndex(index, ListView.Contain)
+                propertiesListView.currentIndex = index
+            }
+        }
+
+        SeparatorLine {}
+
+        RowLayout {
+            id: bottomPanel
+
+            Layout.alignment: Qt.AlignBottom
+            spacing: 12
+            Layout.preferredHeight: 30
+
+            property NavigationPanel navigationPanel: NavigationPanel {
+                name: "ProjectPropertiesBottomPanel"
+                section: root.navigationSection
+                direction: NavigationPanel.Horizontal
+                order: 3
+            }
+
+            FlatButton {
+                text: qsTrc("project", "New property")
+                onClicked: projectPropertiesModel.newProperty()
+
+                navigation.name: "NewProperty"
+                navigation.panel: bottomPanel.navigationPanel
+                navigation.column: 1
+            }
+
+            Item { Layout.fillWidth: true }
+
+            FlatButton {
+                text: qsTrc("global", "OK")
```

Comment:
Button order should be Cancel - OK
Not OK - Cancel

This is how it is in our design

Reply:
And about navigation:
Ok button must be first - navigation.column = 1
Cancel - navigation.column = 2
New property - navigation.column = 3

Reply:
Button order depends on platform, Mac is different from Windows here!

Reply:
Use QtButtonBox instead and let Qt deal with it

Reply:
> Use QtButtonBox instead and let Qt deal with it

I don't think so. This is not the subject of this PR. We'll deal with it later globally throughout the project. Now we need to keep the order of these buttons as in other dialogs

Reply:
> And about navigation: Ok button must be first - navigation.column = 1 Cancel - navigation.column = 2 New property - navigation.column = 3

Are you sure these are the correct numbers? 
![image](https://user-images.githubusercontent.com/88136054/158552441-6bf1f192-ecde-4f6c-84b5-55dacf284d40.png)
Because it goes backward when you press the left arrow key
Did you mean those numbers with the order in which I initially placed them

Reply:
> Because it goes backward when you press the left arrow key

Sorry, I meant 
Ok = 1 
Cancel = 3 
New property = 2